src/screens/OrganizationPeople/OrganizationPeople.spec.tsx

```
@@ -1340,4 +1341,100 @@ describe('OrganizationPeople', () => {
+      }) => (
+        <div>
+          <div data-testid="row-count">{rows?.length ?? 0}</div>
+          <button
+            data-testid="trigger-forward"
+            onClick={() =>
+              gridProps?.onPaginationModelChange?.({
+                page: 1,
+                pageSize: PAGE_SIZE,
+              })
+            }
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)